isolate_example/README.md

```diff
@@ -0,0 +1,16 @@
+# isolate_example
```

Comment:
Like I said offline, you'll want to update this to match the other README files for the other examples.